packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/List.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useMemo } from 'react';
+import { Font } from '@samuelmeuli/font-manager';
+import { List, SearchFonts, FontLI, FontFamily } from './elements';
+
+type Props = {
+  fonts: Font[];
+  onSelection: (e: any) => void;
+  activeFontFamily: string;
+  expanded: boolean;
+};
+
+const FontList = ({
```

Comment:
Make this a `FunctionComponent` for better IDE support.
Also export as default (see @Saeris' rules he's following in other PRs)

```ts
export const FontList: FunctionComponent<Prop> = ({
  // ...
}) => {
  // ...
};
```